src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMoveBookingIntTest.kt

```diff
@@ -305,3 +426,124 @@ fun nomisPhysicalAttributesToOffenderRemeasured(
       ),
     ),
   )
+
+fun nomisProfileDetailsFromOffender(
+  offenderNo: String,
+  bookingStartTime: LocalDateTime = LocalDateTime.now().minusDays(1),
+) =
+  PrisonerProfileDetailsResponse(
+    offenderNo,
+    listOf(
+      BookingProfileDetailsResponse(
+        bookingId = 2345L,
+        startDateTime = "${bookingStartTime.minusDays(8)}",
+        endDateTime = "${bookingStartTime.minusDays(6)}",
+        latestBooking = true,
+        profileDetails = listOf(
+          ProfileDetailsResponse(
+            type = "BUILD",
+            code = "MEDIUM",
+            createDateTime = "${bookingStartTime.minusDays(7)}",
+            createdBy = "A_USER",
+            modifiedDateTime = null,
+            modifiedBy = null,
+            auditModuleName = "NOMIS",
+          ),
+        ),
+      ),
+    ),
+  )
```

Comment:
We only have happy path tests at the moment - next step is to improve the test data builders so other test scenarios are easier to write and comprehend, then I'll get onto some negative testing.